pillar.example

```diff
@@ -1,9 +1,10 @@
 # vim: ft=sls
-ssh-tunnel:
-  remote-hosts:
-    - example1.com:443
-    - example2.com:443
-  users:
-    adam.jensen:
-      name: Adam Jensen
-      pubkey: ssh-ed25519 AAAAC.... Adam Jensen <adam@deusex.universe>
+
+ssh-tunnels:
```

Comment:
Like this as well, not having any additional key. But, I think we need to keep `ssh-tunnel`, as this should match the formula name I think :)

Reply:
Ah right, fixed! thinking further about this, I think we can also make it explicit that it actually creates users or can create users if they do not exist, it is not purely ssh-tunnel configuration .. maybe just in README?

Reply:
Yes, good idea. I think this is not super clear so far :)
Would you mind adding a line there?

Reply:
Added! 👍